src/components/catalogs/CatalogSearch.jsx

```diff
@@ -75,6 +99,51 @@ function CatalogSearch(intl) {
       setSelectedSuggestedCourseType('course');
     }
   };
+  
+  useEffect(() => {
+    // Grab content type(s) or default to course & program
+    let contentToDisplay = contentType ? contentType.slice() : [CONTENT_TYPE_COURSE, CONTENT_TYPE_PROGRAM];
```

Comment:
nit: While slice() is a perfectly valid way to make a shallow copy, my general preference is to use an operator that does that specific operation, rather than a function built to do a different operation (but will do the operation you want if you omit certain arguments). 
```suggestion
    let contentToDisplay = contentType ? [...contentType] : [CONTENT_TYPE_COURSE, CONTENT_TYPE_PROGRAM];
```

Reply:
love this!